openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -25,6 +25,12 @@ class CourseOverview(django.db.models.Model):
     a course as part of a user dashboard or enrollment API.
     """
 
+    # IMPORTANT: Bump this whenever you modify this model and/or add a migration.
+    VERSION = 1
```

Comment:
There are 16 servers, each with lots of gunicorn servers.  Which one would clear it?  I was going to suggest the git inspection, but felt it was a terrible suggestion.